### ISSUE TITLE
Code navigation is not working

### ISSUE THREAD

**Issue by drizt** [eclipse/pdt#88]
**Describe the bug**
Code navigation is not working in PHP Project from Compose Package. I create a new project from yiisoft/yii2-app-basic project, open a model file and try to click on `use Yii` with Ctrl. No effect.

**Describe the eclipse environment**
Eclipse Version: 2020-06 (4.16.0)
PDT 7.2.0.2020052718
No any extra plugins just Eclipse PHP downloaded from a site.

**Describe your system**
 - OS: Fedora
 - Version 31
 - PHP Version 7.3.20
 - Compose 1.10.9

**To Reproduce**
1. Create a new **PHP Project from Composer Package**. As a starting point use yiisoft/yii2-app-basic. Don't use existing location.
2. Open models->ContactForm.php file from project tree.
3. Click on `use Yii` with Ctrl.
4. See no effect.

**Expected behavior**
Opened Yii.php file in new tab.

**Screenshots**
https://i.imgur.com/TzR4kou.png
![image](https://user-images.githubusercontent.com/534665/89178616-722c9300-d5a7-11ea-942c-0854b3f3518c.png)

**Additional context**
It works when I use the same yii project but downlod manually with composer. In this case I use just **PHP Project**. But in this case navigation only works in vendor dir. After I played with Build paths I can get navigation in *controllers/SiteController.php*, *models/ContactForm.php* and *models/User.php*. But still no code navigation in *models/LoginForm.php*. 

Also I can't find info which explains how to setup lookup of include files. I mean *.php file contains `use app\models\MyModel` for example. How PDT can know that `app` means project dir and it must load MyModel.php from /path/to/project/models/ dir?

P.S. I never used Eclipse before and just try to use IDE for php coding.

**Comment by drizt** [comment]
I can found some docs and setup my Build Paths. So now I sure that current settings is correct but Code Assist is not working. New screenshot 
![image](https://user-images.githubusercontent.com/534665/89187507-d30f9780-d5b6-11ea-9416-3a5b1eb20f75.png)

I added only *yiisoft*, *models* and *controllers* to Build Paths. All files in these folders must be parsed on Build Project click. Is it correct? But only files in *yiisoft* dir are visible from *models/\*.php* or *controllers/\*.php* files.

**Comment by fglueck** [comment]
I have the sampe problem with an upgraded version and some plugins and also with a new installed version from eclipse.org PDT Version without any other plugins.

**Comment by fglueck** [comment]
I can fix the problem by:
Menu: Project -> Properties 
search: build
PHP -> Source Paths -> Build Path 
[Add Folder]
here I select the "root" Path for the project. After build process, it works!

**Comment by drizt** [comment]
I tried the same but it's not work for all files.

**Comment by fglueck** [comment]
Yes, I have the same Problem, but after going to the preferences I saw that there my "root" folder is not listen any more. I add it again and now it works for alle files (today, tomorrow I know more  :-) )

**Comment by pounard** [comment]
I happen to regularily experience similar problems, there are a few mystic bugs hanging around I guess. Solution that almost always work to dix it: 1) remove all code and include path 2) restore them 3) go to the "clean..." option ans force your project rebuild. Whenever it does'nt work, delete the project and recreate it. That's my 2c tricks, it always work. One way or thé other, project's indexes always end up broken for some esoterical reasons (not top often that's barely annoying).
